docker/node/Dockerfile

```diff
@@ -1,6 +1,7 @@
-FROM electronuserland/builder:base
+ARG IMAGE_VERSION=base
+FROM electronuserland/builder:$IMAGE_VERSION
 
-ARG NODE_VERSION 18.16.0
+ARG NODE_VERSION 18.15.0
```

Comment:
@mmaietta Is there a reason why the node version was rolled back?

Reply:
Another user commented that the node version needed to be lower?

Could probably just up date it to latest 18.x. Would you mind opening a PR for that change?

Reply:
PR has been raised https://github.com/electron-userland/electron-builder/pull/7787